packages/feedback/src/FeedbackButton.js

```diff
@@ -16,7 +16,11 @@ const FeedbackButton = ({
   <Button
     style={btnStyles}
     color={active === icon ? 'primary' : 'light'}
-    onClick={() => onClick(icon)}
+    aria-pressed={active === icon}
+    onClick={() => {
+      onClick(icon);
+    }}
+    onKeyDown={({ keyCode }) => keyCode === 13 && onClick(icon)}
```

Comment:
Does `onKeyDown` only fire when the Button is focused or is it listening whenever the Button is on the screen?

Reply:
It's only when the key is focused (from what I tested and see online)